test/test_congress.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+import responses
+import os
+import json
```

Comment:
Why don't we use `simplejson` here for a performance boost. (Just a demonstration of reviewing -- not meant to be acted upon; `json` is a good choice since it's part of the standard library.)